roles/backend/templates/backend.yml.j2

```diff
@@ -3,17 +3,9 @@
 # {{ managed }}
 #
 
-##
-# Sensu backend configuration
-##
-{# -------------------------------------- #}
-{% if backend_debug %}
-debug: {{ backend_debug }}
-{% endif %}
-{# -------------------------------------- #}
-{% if backend_log_level %}
-log-level: {{ backend_log_level }}
-{% endif %}
-{# -------------------------------------- #}
 state-dir: "/var/lib/sensu/sensu-backend"
-{# -------------------------------------- #}
+config-file: "/etc/sensu/backend.yml"
```

Comment:
Does this line make any difference? Once it is interpreted, Sensu already finished reading the configuration and I would be surprised to see it restart the configuration process.

If we would like to support using a custom configuration file, we should probably pass it as a command-line parameter. But this means that the service that is responsible for controlling the Sensu backend must support passing custom flags to the executable.

I would say that we remove this line and be done with it.

Reply:
👍 
I've left it in because somehow I thought it was funny in how it self-references, even in their example config file. I have not thought of an adverse effect of the deamon restarting (indefinitely?).